package.json

```diff
@@ -71,6 +71,7 @@
     "wordwrap": "1.0.0"
   },
   "devDependencies": {
+    "assert": "^1.4.1",
```

Comment:
https://www.npmjs.com/package/assert adds support to run the assert tests in the browser.